README.rst

```diff
@@ -47,13 +47,54 @@ is a `template`_ you can use.
 Create a GoogleAdsClient
 ########################
 
+Using yaml file
```

Comment:
s/yaml/YAML

Reply:
A note here that `s/yaml/YAML` may not be clear, and means: _replace `yaml` with `YAML`_.

I don't think it's worth holding this PR up for these changes. I can make them in a later release.

Reply:
Fixed